src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlAttributes.cs

```diff
@@ -74,6 +75,7 @@ internal XmlAttributeFlags XmlFlags
 
         private static Type IgnoreAttribute
         {
+            [RequiresUnreferencedCode("calls GetType")]
```

Comment:
Looking around, I can't find a `System.XmlIgnoreMemberAttribute` anywhere. Can the code that is looking for it be removed?

Reply:
Looking at [https://referencesource.microsoft.com/#mscorlib/system/xmlignorememberattribute.cs,33ba544d60bd65ba,references](https://referencesource.microsoft.com/#mscorlib/system/xmlignorememberattribute.cs,33ba544d60bd65ba,references), it appears it was only used for xml serializing AppDomainSetup. But that no longer appears to be the case in .NET Core.

---
In reply to: [595310314](https://github.com/dotnet/runtime/pull/49682#discussion_r595310314) [](ancestors = 595310314)

Reply:
Ah, and since it's fetched from corlib this should always return null, right?

Reply:
I've opened https://github.com/dotnet/runtime/issues/49761 for this. I'm not sure if we want to do this in your PR or separately. But, it looks like it will affect what you are marking as `RUC` here.